rust/lance/src/index/vector/ivf/v2.rs

```diff
@@ -414,19 +410,6 @@ impl<S: IvfSubIndex + fmt::Debug + 'static, Q: Quantization + fmt::Debug + 'stat
         self.ivf.find_partitions(&query.key, query.nprobes, dt)
     }
 
-    // async fn append(&self, batches: Vec<RecordBatch>) -> Result<()> {
-    //     IvfIndexBuilder::new(
-    //         dataset,
-    //         column,
-    //         index_dir,
-    //         distance_type,
-    //         shuffler,
-    //         ivf_params,
-    //         sub_index_params,
-    //         quantizer_params,
-    //     )
-    // }
-
```

Comment:
Remove?

Reply:
yeah these lines are commented and not used, so just removed them